snackpub/.gitignore

```diff
@@ -0,0 +1,10 @@
+*~
+.DS_Store
+
+npm-debug.log
+node_modules/
+
+logs/*.log
+tmp/*
+
+build/
```

Comment:
Can we consolidate this in the root `.gitignore`? E.g. the `npm-debug.log` and `node_modules` should already be covered by the root `.gitignore`. Just feels a bit cleaner vs adding these files to every workspace (unless it's doing some really special things ofc)